examples/CRISP/package.json

```diff
@@ -1,75 +1,36 @@
 {
   "name": "crisp",
-  "version": "0.0.0",
-  "description": "CRISP - Coercion-Resistant Impartial Selection Protocol",
-  "repository": "https://github.com/gnosisguild/enclave",
+  "private": true,
   "type": "module",
+  "description": "CRISP - Coercion-Resistant Impartial Selection Protocol",
   "author": {
     "name": "gnosisguild",
     "url": "https://github.com/gnosisguild"
   },
   "scripts": {
-    "compile": "forge compile",
-    "ciphernode:add": "hardhat ciphernode:add",
```

Comment:
do we want to keep these here and call them within the contracts folder? so one can run from the CRISP root too